packages/payment-proxy-client/src/file.ts

```diff
@@ -5,15 +5,18 @@ export async function fetchFile(
   url: string,
   paymentAmount: number,
   channelId: string,
-  nitroClient: NitroRpcClient
+  nitroClient: NitroRpcClient,
+  updateChannelCallback: () => void
```

Comment:
I'm not fussed about how this is implemented just now / open to suggestions for a different approach.